Shadowlands/TheNecroticWake/Trash.lua

```diff
@@ -417,6 +427,25 @@ function mod:ZolramusBonemenderDeath(args)
 	self:ClearNameplate(args.destGUID)
 end
 
+-- Zolramus Sorcerer
+
+function mod:ShadowWell(args)
+	if self:Friendly(args.sourceFlags) then -- these NPCs can be mind-controlled by Priests
+		return
+	end
+	self:Message(args.spellId, "red", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+	self:Nameplate(args.spellId, 0, args.sourceGUID)
+end
+
+function mod:ShadowWellSuccess(args)
+	self:Nameplate(args.spellId, 13.6, args.sourceGUID)
```

Comment:
From my logs I have this at 13.1. But you also need to subtract the 1s cast time because the timer is started in `SPELL_CAST_SUCCESS` and we want to track the time left until the next time we will alert for it which is in `SPELL_CAST_START`. So 12.1 seems correct to me.

Reply:
If with the earlier comment I don't start the Nameplate, wouldn't that mean that the nameplate with the value would also not really be useful? Starting a bar timer would then make more sense right?

Reply:
In general we want timers to end when the LittleWigs alert for the spell is shown. But in cases where the alert is shown before the ability goes on cooldown then it will just display as "off cooldown" for that 1s. There's just no need to indicate that on the first cast.

This won't really be an issue anymore when we can start timers when mobs enter combat, but that functionality isn't ready just yet.

We don't use a bar timer because there's more than one mob which casts the ability.